### ISSUE TITLE
any media object additions for media editor?

### ISSUE THREAD

**Issue by nconrad** [ModelSEED/ProbModelSEED#5]
@janakagithub, @jplfaria , @samseaver.   It may be a good time to get all the data we need for editing media into the media objects.  Can we have the media name added?  I think it makes sense to have the name readily available, particularly if custom compounds will be supported?  Anything else?  Charge? Below is all the data I have right now.  

![screen shot 2015-06-01 at 3 51 55 pm](https://cloud.githubusercontent.com/assets/1194246/7923243/23a9a00e-0877-11e5-99ab-68bf8bd09fe5.png)

**Comment by jplfaria** [comment]
Compound names are essential. People will want to search for compound names, not IDs.

**Comment by cshenry** [comment]
Don�t need charge, but name makes alot of sense. I can add that.

**Comment by janakagithub** [comment]
Yes, compounds should be searchable by their aliases and abbreviations.

Sent from my Google Nexus 5
Don�t need charge, but name makes alot of sense. I can add that.

On Jun 1, 2015, at 4:05 PM, jplfaria <notifications@github.com> wrote:

> Compound names are essential. People will want to search for compound
names, not IDs.
>
> �
> Reply to this email directly or view it on GitHub.
>

—
Reply to this email directly or view it on GitHub
<https://github.com/ModelSEED/ProbModelSEED/issues/5#issuecomment-107720916>
.

**Comment by chian** [comment]
Can we add blank spot for pH when we need it? May come in handy at some point.

**Comment by cshenry** [comment]
I think it's already there, but if it's not, I'll add it.

**Comment by mdejongh** [comment]
It would be good to search using identifiers from other databases too,
e.g., KEGG compound ids.

Matt

On Mon, Jun 1, 2015 at 8:08 PM, cshenry <notifications@github.com> wrote:

> I think it's already there, but if it's not, I'll add it.
>
> Sent from my iPhone
>
> > On Jun 1, 2015, at 6:26 PM, chian <notifications@github.com> wrote:
> >
> > Can we add blank spot for pH when we need it? May come in handy at some
> point.
> >
> > Sent from my iPhone
> >
> > > On Jun 1, 2015, at 5:44 PM, janakagithub <notifications@github.com>
> wrote:
> > >
> > > Yes, compounds should be searchable by their aliases and abbreviations.
> > >
> > > Sent from my Google Nexus 5
> > > Don�t need charge, but name makes alot of sense. I can add that.
> > >
> > > On Jun 1, 2015, at 4:05 PM, jplfaria <notifications@github.com> wrote:
> > >
> > > > Compound names are essential. People will want to search for compound
> > > names, not IDs.
> > > >
> > > > �
> > > > Reply to this email directly or view it on GitHub.
> > > >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> https://github.com/ModelSEED/ProbModelSEED/issues/5#issuecomment-107720916
> >
>
> > > .
> > > —
> > > Reply to this email directly or view it on GitHub.
> > >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ModelSEED/ProbModelSEED/issues/5#issuecomment-107750528>
> .
>



-- 
Matthew DeJongh, Ph.D.
Computer Science Department
Hope College, Holland, MI 49423
(616) 395-7429

**Comment by samseaver** [comment]
I think formula would be useful, even if its self-explanatory in many cases, it serves as a reminder of the quantity of individual elements that would be consumed.

**Comment by mmundy42** [comment]
@nconrad, what else do you need here?  It wasn't clear to me what server method is being used for media editing.

**Comment by nconrad** [comment]
The media editor above was prototyping with KBase data.  Here's an update on modelseed/patric media.  

I added a list of Chris's public media and media tables to the site under biochemistry.  Calling get object on one of these returns a string of a tab delimited table.  It currently has:

- compound name
- compound id
- concentration
- min flux
- max flux

Next steps/questions.

1) @chian requests that pH is added
2) @mdejongh requests that KEGG ids are added 
3) Where should we store user's media?  /username/Media?
4) Since the media table/object is simple and small, editing the workspace data should be fine here.  Any other ideas? 
5) How can the user select their own media for reconstruction?  I was thinking of just adding a radio button in the form to switch between public and user's media.  Sound good?
